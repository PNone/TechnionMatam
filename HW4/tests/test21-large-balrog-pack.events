PotionsMerchant PotionsMerchant PotionsMerchant PotionsMerchant
Pack 12 Balrog Balrog Balrog Balrog Balrog Balrog Balrog Balrog Balrog Balrog Balrog Balrog