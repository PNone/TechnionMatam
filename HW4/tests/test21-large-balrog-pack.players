Daniel Warrior Responsible
Regev Magician RiskTaking
Baraa Archer RiskTaking
Baraako Magician RiskTaking
Sel Magician RiskTaking
Mai Warrior RiskTaking