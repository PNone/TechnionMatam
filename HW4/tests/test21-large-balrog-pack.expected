Game is about to start!
These are the players:

1. Daniel, Warrior with Responsible character (level 1, force 5)
2. Regev, Magician with RiskTaking character (level 1, force 5)
3. Baraa, Archer with RiskTaking character (level 1, force 5)
4. Baraako, Magician with RiskTaking character (level 1, force 5)
5. Sel, Magician with RiskTaking character (level 1, force 5)
6. Mai, Warrior with RiskTaking character (level 1, force 5)

----------------------------------------

New round starts!

Turn: 1
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 2
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 3
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Turn: 4
Player: Baraako, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraako bought 0 potions!

Turn: 5
Player: Sel, Magician with RiskTaking character (level 1, force 5)
Event: Pack of 12 members (power 180, loot 1200, damage 108012)
Sel lost the encounter and took 108012 damage!

Turn: 6
Player: Mai, Warrior with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Mai bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     100   10
3          Daniel          1         5     150   10
4          Mai             1         5     150   10
5          Regev           1         5     100   10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 7
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 8
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 9
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Turn: 10
Player: Baraako, Magician with RiskTaking character (level 1, force 5)
Event: Pack of 12 members (power 204, loot 1200, damage 108012)
Baraako lost the encounter and took 108012 damage!

Turn: 11
Player: Mai, Warrior with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Mai bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     150   10
5          Regev           1         5     100   10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 12
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 13
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 14
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Turn: 15
Player: Mai, Warrior with RiskTaking character (level 1, force 5)
Event: Pack of 12 members (power 228, loot 1200, damage 108012)
Mai lost the encounter and took 108012 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     100   10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 16
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 17
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Regev bought 0 potions!

Turn: 18
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     100   10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 19
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 20
Player: Regev, Magician with RiskTaking character (level 1, force 5)
Event: Pack of 12 members (power 252, loot 1200, damage 108012)
Regev lost the encounter and took 108012 damage!

Turn: 21
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 22
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 23
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: PotionsMerchant
Baraa bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     100   20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 24
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Turn: 25
Player: Baraa, Archer with RiskTaking character (level 1, force 5)
Event: Pack of 12 members (power 276, loot 1200, damage 108012)
Baraa lost the encounter and took 108012 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 26
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 27
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 28
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 29
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: PotionsMerchant
Daniel bought 0 potions!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     150   10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

New round starts!

Turn: 30
Player: Daniel, Warrior with Responsible character (level 1, force 5)
Event: Pack of 12 members (power 300, loot 1200, damage 108012)
Daniel lost the encounter and took 108012 damage!

Round has ended!
Leader board is as follows:
Ranking    Player Name     Level     Force HP    Coins
1          Baraa           1         5     0     20
2          Baraako         1         5     0     10
3          Daniel          1         5     0     10
4          Mai             1         5     0     10
5          Regev           1         5     0     10
6          Sel             1         5     0     10

----------------------------------------

Game is over!
There are no winners in this game.
Better luck next time!
